lib/Doctrine/Common/DataFixtures/Executor/AbstractExecutor.php

```diff
@@ -141,6 +141,7 @@ public function purge()
      *
      * @param array $fixtures Array of fixtures to execute.
      * @param boolean $append Whether to append the data fixtures or purge the database before loading.
+     * @param boolean $single_transaction Whether to use a single transaction when loading fixtures
      */
-    abstract public function execute(array $fixtures, $append = false);
+    abstract public function execute(array $fixtures, $append = false, $single_transaction = true);
```

Comment:
I know at least one project, that extends the executor, so this may lead to unexpected behaviour (BC). One can pass `false` as second argument, but the concrete executor class ignores it.